doc/api/util.md

```diff
@@ -75,7 +75,7 @@ is wrapped in an `Error` with the original value stored in a field named
 added: v0.11.3
 -->
 
-* `section` {string} A string identifying the portion of the application for
+* `section` {string} A wildcard string identifying the portion of the application for
```

Comment:
The string itself is not a wildcard string as long as there is no asterisk included.

Reply:
thx, updated.